src/inlineeditorui.js

```diff
@@ -175,4 +193,16 @@ export default class InlineEditorUI extends EditorUI {
 			} );
 		}
 	}
+
+	/**
+	 * Set max-width for toolbar.
+	 *
+	 * @private
+	 */
+	_setToolbarMaxWidth( width ) {
```

Comment:
This method is super small. There's no point in keeping it. Let's move these few lines directly to the callback.

Reply:
Sure, I'll remove it.